concordium-node/src/bin/collector.rs

```diff
@@ -23,6 +23,9 @@ use std::alloc::System;
 #[global_allocator]
 static A: System = System;
 
+// The timeout for the RPC calls.
+const GRPC_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
This does look plausible, but you you please make this a configurable option with 30s default, like the other options we have?